cloud/cluster/node.go

```diff
@@ -1,32 +1,53 @@
 package cluster
 
+import (
+	"fmt"
+)
+
 type NodeId string
 
 type Node interface {
```

Comment:
Y'know, I'm not sure yet, but I'm starting to wonder what other implementations of Node there will be.... (Just an idle thought)